prime/src/components/borrow/ManageAccountWidget.tsx

```diff
@@ -176,6 +193,11 @@ export default function ManageAccountWidget(props: ManageAccountWidgetProps) {
   const [errorMsg, setErrorMsg] = useState<string | undefined>(undefined);
   // modals
   const [showAddActionModal, setShowAddActionModal] = useState(false);
+  const actionCardRefs = useRef<(HTMLDivElement | null)[]>([]);
+
+  useEffect(() => {
+    actionCardRefs.current = actionCardRefs.current.slice(0, activeActions.length);
```

Comment:
I don't understand how this works and what `current` refers to, but I'll trust you on it.

Reply:
Is this just trimming down the array when actions are removed?

Reply:
It synchronizes the length of actionCardRefs (used for scrolling to specific action cards) to the number of active actions. `current` is a special property of the `useRef` hook.